File: Backend/TravellifeChaser/Models/AirlinesSystem/Pricelist.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TravellifeChaser.Models
{
    public class Pricelist
    {
        public int Id { get; set; }
        public double LuggageOver10kg { get; set; }
        public double LuggageOver20kg { get; set; }
        public double HandLuggageOverMaxDimensions { get; set; }

        public Airline Airline { get; set; }  // 1 : 1 Airline - Pricelist
    }
}

File: AvioAndRentACarSystem/src/app/AirlineModel/priceList.ts
export class PriceList
{
    id: number;
    luggageOver10kg: number;
    luggageOver20kg: number;
    handLuggageOverMaxDimensions: number;

    constructor(luggageOver10kg: number, luggageOver20kg: number, handLuggageOverMaxDimensions: number)
    {
        this.luggageOver10kg = luggageOver10kg;
        this.luggageOver20kg = luggageOver20kg;
        this.handLuggageOverMaxDimensions = handLuggageOverMaxDimensions;
    }
}

export class XYZ
{
    X : number;
    Y : number;
    Z : number;

    constructor(X : number, Y : number, Z : number)
    {
        this.X = X;
        this.Y = Y;
        this.Z = Z;
    }
}
